src/repositories/audit.repository.ts

```diff
@@ -3,7 +3,9 @@ import {juggler, DefaultCrudRepository} from '@loopback/repository';
 
 import {AuditLog} from '../models';
 import {AuditDbSourceName} from '../types';
+import {tenantGuard} from '@sourceloop/core';
 
+@tenantGuard()
```

Comment:
create, update and delete is already handled via mixin and we dont have any read operation in this 
do we really need this

Reply:
AuditLogRepository and AuditLog Model of loopback4-audit-log and @sourceloop/audit-service should be same, otherwise they causes error.
![image](https://github.com/sourcefuse/loopback4-audit-log/assets/107617248/48f4e47c-5381-4c01-9b94-0f8434889f3b)

will test on monday after removing tenatguard from here, if everything works fine, then will remove it.

Reply:
removed